lib/docx/elements/element.rb

```diff
@@ -14,13 +14,20 @@ def self.included(base)
       end
 
       attr_accessor :node
-      delegate :at_xpath, :xpath, :to => :@node
 
       # TODO: Should create a docx object from this
       def parent(type = '*')
         @node.at_xpath("./parent::#{type}")
       end
 
+      def at_xpath(*args)

```

Comment:
arguments match what is expected in the [docs](https://www.rubydoc.info/github/sparklemotion/nokogiri/Nokogiri%2FXML%2FSearchable:at_xpath)